samples/Chat/src/app/ConfigurationScreen.tsx

```diff
@@ -175,8 +175,10 @@ export default (props: ConfigurationScreenProps): JSX.Element => {
             {PROFILE_LABEL}
           </Text>
           <div className={largeAvatarContainerStyle(selectedAvatar)}>
-            <div aria-label={`${selectedAvatar} avatar`} className={largeAvatarStyle}>
-              {selectedAvatar}
+            <div aria-label={`${selectedAvatar} avatar selected`} aria-live="polite" className={largeAvatarStyle}>
+              <div aria-label="" aria-hidden="true">
```

Comment:
you shouldn't need to make the aria-label a empty string here. aria-hidden should make the narrator ignore it. otherwise looks good!